api/controllers/chefs-api-controller.js

```diff
@@ -0,0 +1,6 @@
+const postTest = async (req, res) => {
+  console.log(req.body);
```

Comment:
slam a TODO on this to remove this console.log in a later PR.

We don't want this going "live" anywhere.

Reply:
Done